.github/workflows/docs.yml

```diff
@@ -71,3 +71,4 @@ jobs:
           path: docs/build
           retention-days: 1
           if-no-files-found: error
+      - uses: ./.github/workflows/preview-docs.yml
```

Comment:
Does this work PRs filed from a fork?

Reply:
I'm trying to figure this out. I switched to this approach because the job wasn't showing up on the PR checks. But maybe it doesn't have to? But it was nice that it referred to the workflow from the PR, not on master.

Re secrets: `secrets: inherit` can be used

And it's failing right now with the following message. According to [the docs](https://docs.github.com/en/actions/how-tos/sharing-automations/reusing-workflows#calling-a-reusable-workflow), it should work. 

```
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/mlflow/mlflow/.github/workflows/preview-docs.yml'. Did you forget to run actions/checkout before running your local action?
```

Reply:
> the job wasn't showing up on the PR checks. But maybe it doesn't have to?

It doesn't have to.